src/templates/legacy_app.ejs

```diff
@@ -5,18 +5,21 @@ window.helpers = helpers;
 window.Base64 = Base64;
 
 <% if (helpers.async) { %>
+const invokeMethods = /^(hide|show|preloadPane|popover|enableSave|disableSave|setIconState|notify)$/
```

Comment:
Any reason not to include this info as metadata in the map?

Reply:
My gut feeling was that it would be simpler this way.  Reading in the metadata, then passing to the `wrapSdkClient` helper would require a refactor to accommodate a `method` param (most likely)